Cargo.lock

```diff
@@ -3062,7 +2649,7 @@ dependencies = [
 [[package]]
 name = "fp-rpc"
 version = "3.0.0-dev"
-source = "git+https://github.com/humanode-network/frontier?branch=locked/polkadot-v0.9.40#ac5c3415f47adf126419579924052daff07301e1"
+source = "git+https://github.com/humanode-network/frontier?branch=locked/polkadot-v0.9.40#a67a3794328c80150f162c51e71646cdc76a65a8"
```

Comment:
The whole idea behind the `locked` branches is that they never change (never pushed to after their creation).

For this we would create a new branch `locked/polkadot-v0.9.40-1` and push there, and once we are done we'd fix the commit we're on.

I guess tags are a much better fit for this kind of version binding actually...